### DISCUSSION TITLE
xtb-python and rdkit question

### DISCUSSION THREAD

**Discussion by darthmodel** [grimme-lab/xtb#686] [General]
Dear developers, 

I wrapped rdkit around xtb-python to process multiple molecules in an input sdf file. The result from `calc.get_energy()` doesn't seem to match with the command line result for the same molecule. I checked `atomicNumbers` and `atomicCoords` against the file. Both seem to be read and set properly. Where could it be wrong? Thanks. 

```
18   │     df = PandasTools.LoadSDF(sdfFileIn, removeHs=False)
19   │ 
20   │     for mol in df.ROMol:
21   │         atomicNumbers = np.array([atom.GetAtomicNum() for atom in mol.GetAtoms()])
22   │         atomicCoords = mol.GetConformer().GetPositions()
23   │ 
24   │         calc = Calculator(Param.GFN2xTB, atomicNumbers, atomicCoords, charge=0)
25   │         calc.set_verbosity(VERBOSITY_MINIMAL)`
```

Best, 
Cui

**Comment by kjelljorner** [comment]
You need to convert from angstrom to Bohr

**Reply by darthmodel** [reply]
Ah, the units! That makes so much sense. Thanks so much!!!
